From: Shyamanth RH <shyamanth.rh@arm.com>
Date: Mon, 17 Aug 2026 00:00:00 +0000
Subject: [PATCH] rdv3: enable the external RME ACS EL3 handler

Build TF-A with the upstream vendor ACS service enabled while retaining
the platform's branch-protection setting for the ACS EL3 relink.

Signed-off-by: Shyamanth RH <shyamanth.rh@arm.com>
---
 configs/rdv3/rdv3 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/rdv3/rdv3 b/configs/rdv3/rdv3
--- a/configs/rdv3/rdv3
+++ b/configs/rdv3/rdv3
@@ -79,7 +79,7 @@ TF_A_ENABLE_SPM=0
 TF_A_RAS_FW_FIRST=0
 TF_A_BL31_IN_DRAM=0
 TF_A_ENABLE_MPAM_FOR_LOWER_ELS=1
-TF_A_BUILD_FLAGS="RME_GPT_BITLOCK_BLOCK=0"
+TF_A_BUILD_FLAGS="RME_GPT_BITLOCK_BLOCK=0 ENABLE_ACS_SMC=1"
 TF_A_PLATFORM_VARIANT=0
 TF_A_BUILD_VARIANT="rdv3"
 TF_A_BUILD_FLAGS="$TF_A_BUILD_FLAGS NRD_PLATFORM_VARIANT=$TF_A_PLATFORM_VARIANT"
-- 
2.43.0
